frontend/src/data/parseReportData.ts

```diff
@@ -174,54 +174,73 @@ export const implementationFromData = (
     dialects: data.dialects.map((uri) => Dialect.forURI(uri)),
   }) as Implementation;
 
-export const parseImplementationData = (
-  loaderData: Record<string, ReportData>,
+/**
+ * Computes a summarized implementation report for the
```

Comment:
```suggestion
 * Compute a summarized implementation report for the
```

Reply:
Should I change it to "Prepare a summarized implementation report...." ?

Reply:
I changed it to "Prepare" since we aren't really computing anything ig. So now the docstring reads as follows:

/**
 * Prepare a summarized implementation report for the
 * passed implementation using all the dialect reports data
 * that was fetched.
 */

Reply:
The verb matters less to me (either's fine) compared to that the docstring is in imperative mood ("Do foo") rather than indicative ("Does foo") but yeah Prepare is good with me.